openapiart/bundler.py

```diff
@@ -1047,9 +1062,18 @@ def _apply_common_x_field_pattern_properties(
             else:
                 schema["format"] = finalised_format
         if "length" in xpattern and int(xpattern["length"]) not in [32, 64]:
+            min_val = (
```

Comment:
for unsigned, range: 0 to (2^N) - 1
for signed, range: -2^(N-1) to 2^(N-1)-1

where N is number of bits or length